sensor_msgs/test/test_pointcloud_conversion.cpp

```diff
@@ -18,7 +18,26 @@
 #include "sensor_msgs/msg/point_cloud.hpp"
 #include "sensor_msgs/msg/point_cloud2.hpp"
 
+#ifdef _MSC_VER
+#pragma warning(push)
```

Comment:
this seem to be pushing the "warning configuration" state, but not  adding any `#pragma warning( disable : ...)` in the middle